build_container/build_container_common.sh

```diff
@@ -1,17 +1,38 @@
 #!/bin/bash -e
 
+function download_and_check () {
+  local to=$1
+  local url=$2
+  local sha256=$3
+
+  curl -fsSL --output "${to}" "${url}"
+  echo "${sha256}  ${to}" | sha256sum --check
+}
+
 if [[ "$(uname -m)" == "x86_64" ]]; then
   # buildifier
-  VERSION=0.28.0
-  SHA256=3d474be62f8e18190546881daf3c6337d857bf371faf23f508e9b456b0244267
-  curl --location --output /usr/local/bin/buildifier https://github.com/bazelbuild/buildtools/releases/download/"$VERSION"/buildifier \
-    && echo "$SHA256  /usr/local/bin/buildifier" | sha256sum --check \
-    && chmod +x /usr/local/bin/buildifier
+  VERSION=0.29.0
+  download_and_check /usr/local/bin/buildifier https://github.com/bazelbuild/buildtools/releases/download/"$VERSION"/buildifier \
+    4c985c883eafdde9c0e8cf3c8595b8bfdf32e77571c369bf8ddae83b042028d6
+  chmod +x /usr/local/bin/buildifier
+
+  # buildozer
```

Comment:
Thanks; if you also add https://github.com/envoyproxy/envoy-build-tools/pull/10/commits/5929083beb1e2ad26bce19fb55aef39d41819a4d we can skip #10.

Reply:
Oh, I see, the release tar ball should have this. LGTM!